python-checks/src/test/resources/checks/duplicatedMethodImplementationCheck.py

```diff
@@ -0,0 +1,160 @@
+class clazz:
+  def method(self):
+    foo()
+    bar()
+
+  def method2(self): #Noncompliant
+#     ^^^^^^^
+    foo()
+    bar()
+
+  def method3(self): #Noncompliant
+#     ^^^^^^^
+    foo()
+    bar()
+
+
+class clazz:
+  def method(self):
+    bar()
+    foo()
+
+  def method2(self):
+    foo()
+    bar()
+
+class clazz:
```

Comment:
What's the intention of this test?

Reply:
It's for coverage completeness on the isException method. I admit it's not the most valuable out there, though...